### PR TITLE
Publish C++ CSS parser

### PR DESCRIPTION
**Pull request by amaltas**
Also fixes some build errors detected while whitelisting parse css library.

### CODE REVIEW

File: validator/htmlparser/css/parse-css-urls_test.cc [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      return "IMAGE_URL";
+    case url::CssSegment::OTHER_URL:
+      return "OTHER_URL";
+  }
+}
+
+// This is so that gunit/gmock print reasonable errors in case a
+// CssSegmentEq match fails. Must be defined in the
+// quality_dni::parse_css_urls namespace, like url::CssSegment itself. See
+// go/gunitadvanced#teaching-googletest-how-to-print-your-values
```
Comment (Gregable) [inline validator/htmlparser/css/parse-css-urls_test.cc:45]: Maybe remove / fix this comment so it makes more sense in github.

Comment (amaltas) [reply validator/htmlparser/css/parse-css-urls_test.cc]: Removed the comment inline in this pull.

**Comment by amaltas** [comment]
@googlebot I fixed it

**Comment by honeybadgerdontcare** [comment]
@googlebot I fixed it

**Pull request merged**